triaging.rst

```diff
@@ -3,193 +3,247 @@
 Triaging an Issue
 =================
 
-When you have the Developer role on the `issue tracker`_ you are able to triage
-issues directly without any assistance.
+This section of the devguide documents the `issue tracker`_ for users
+and developers. 
 
-Fields
-------
+Contributors with the Developer role on the `issue tracker`_ can triage issues
+directly without any assistance.
+
+Fields in the Issue Tracker
+---------------------------
+
+The major elements found in an issue report include:
+
+* Classification (including *Title*) - These fields categorize the issue.
+  The fields include *Title*, *Type*, *Stage*, *Components*, and *Version*.
+* Process - These fields indicate the state of the issue and its progress
+  toward resolution. The fields are *Status*, *Resolution*, *Dependencies*,
+  *Superseder*, *Assigned To*, *Nosy List*, *Priority*, *Keywords*, *Comment*,
+  *File*, *File Description*, *Remote hg repo*, *GitHub PR*.
+* Messages
+* History
 
 Title
 '''''
-Should be properly descriptive of what the issue is about. Occasionally
-people file an issue that either has too generic of a title or end up thinking
-they filed about X but in fact it turns out to be about Y and thus the
-title is now wrong.
+A brief description of the issue. Review whether the title is too generic or
+specifies an incorrect term or library.
+
+Adding a prefix at the start of the title to indicate the module, such as 
+IDLE, DOC, or ASYNCIO, is optional, but can be helpful to others.
```

Comment:
For most modules, and even `doc` as a pseudo-module, I'd suggest a lower case prefix - caps are kinda shouty. (`IDLE` is an exception, since the IDE itself actually is called `IDLE`)

Reply:
Going to fix this as a I see a typo as well.